urlwatch

```diff
@@ -370,6 +394,19 @@ if __name__ == '__main__':
             if display_errors:
                 details += foutput('error', job, (repr(error) +
                         '\n' + str(error)).strip(), summary)
+        except OSError, error:
+            log.error('got OSError while trying to execute diff command: %s' % error)
```

Comment:
The `log.error()` function does `%`-formatting by itself, so `log.error('got ... %s', error)` would be better here (and below).